lib/Service/ImportService.php

```diff
@@ -90,21 +88,22 @@ public function importArticles(string $userId, array $json): ?\OCP\AppFramework\
         // if the feed does not exist, create a separate feed for them
         foreach ($json as $entry) {
             $item = Item::fromImport($entry);
-            $feedLink = $entry['feedLink'];  // this is not set on the item yet
+            $feedLink = $entry['feedLink'];  // this is not set on the item
 
             if (array_key_exists($feedLink, $feedsDict)) {
                 $feed = $feedsDict[$feedLink];
             } else {
+                $this->logger->info("Creating new feed for import of". $feedLink);
```

Comment:
This is the cleaner way.
```suggestion
                $this->logger->info("Creating new feed for import of {url}", ['url' => $feedLink]);
```

Reply:
Thanks :)